React/main/src/app/components/App/index.js

```diff
@@ -1,18 +1,32 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { Provider } from 'react-redux';
 
 import Game from '~screens/Game';
 
 import setupStore from '~/../redux/store';
 
+import WinningMovesService from '../../../services/WinningMovesService';
+
 const store = setupStore();
 
-function App() {
-  return (
-    <Provider store={store}>
-      <Game />
-    </Provider>
-  );
+class App extends Component {
+  state = {
+    moves: []
+  };
+
+  async componentWillMount() {
```

Comment:
ComponentWillMount is deprecated in the last version of React. You can change it and use componentDidMount